packages/contracts/src/framework/dao/DAOFactory.sol

```diff
@@ -87,7 +89,7 @@ contract DAOFactory is ERC165, ProtocolVersion {
         createdDao = _createDAO(_daoSettings);
 
         // Register DAO.
-        daoRegistry.register(createdDao, msg.sender, _daoSettings.subdomain);
+        daoRegistry.register(IDAO(createdDao), msg.sender, _daoSettings.subdomain);
```

Comment:
why is this typecast necessary? `DAO.sol` implements `IDAO` and thus this shouldn't be necessary

Reply:
While I was doing the split, I had `IDAO` present in `osx` and `osx-commonts-contracts` so the solidity compiler complaint. Since we now only have one `IDAO`, you are right that it can be removed again. Reverted in a3e9c63302ea1c71fc897fe07a81f8edadf71c74

Good catch!